tests/cli/fluvio_smoke_tests/topic-basic.bats

```diff
@@ -23,7 +23,7 @@ setup_file() {
     TOPIC_NAME_SYSTEM=$(random_string)
     export TOPIC_NAME_SYSTEM
 
-    DEDUP_FILTER_NAME="dedup-filter"
+    DEDUP_FILTER_NAME="dedup-bloom-filter"
```

Comment:
Is there integration or CLI test?

Reply:
this is just a test creating a sm. I'll create cli tests for the "--dedup" then

Reply:
I added a cli test

Reply:
Actually, I removed this test, as it's needed to be logged to download this SM